core/src/main/java/org/apache/accumulo/core/logging/Logging.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.accumulo.core.logging;
+
+public class Logging {
+  public static final String PREFIX = "org.apache.accumulo.logs.";
```

Comment:
"logs" here is either redundant (if it refers to the fact that this is a log message), or ambiguous (if it refers to WAL or some other kind of logging).

Is the intent to have this used for specific kinds of logs, or replace all Accumulo logging? If it is the former, the name might need to be more specific to reflect the category of logs it represents.

Reply:
My goal is to apply a logical structure to logging namespace based on function that cross cuts lots of code, allowing easily adjusting log levels based on function by users.  Also, centralizing important log messages in the source code makes it easier for user interested in automatic parsing of log messages to track changes. I don't think all logs would go here.

For the name, I was thinking `org.apache.accumulo.logs.` would be the prefix for everything in this logical logging namespace.  Under it would have things like `org.apache.accumulo.logs.fate`, `org.apache.accumulo.logs.metadata`, `org.apache.accumulo.logs.tablet`, etc.

At first I tried using something like `accumulo.tablet` for the logical namepsace, but I found this annoying when I tried to turn on debug logging for Accumulo because I had to set two things in the log4j config (`log4j.logger.org.apache.accumulo=DEBUG` and `log4j.logger.accumulo=DEBUG`).  Based on this I decided it would be nice to share the prefix `org.apache.accumulo`  with class loggers, but then add something that is different than all existing packages.  Any suggestions for the namespace prefix would be appreciated.

Reply:
I like the idea of logical logging, but I don't think we need a global prefix for it to separate it from code-based logging with a separate namespace. It's probably better to just remove the `.logs` part of the prefix. Over time, more logging should be logical and less code-based, so the question is moot in the long term (because `org.apache.accumulo` will eventually be the top-level logical logging prefix), and unless there's a conflict, it's not really an issue in the short-term. Using just `org.apache.accumulo` as the prefix aligns with the current practice of using `org.apache.accumulo.audit` as the logical logging prefix for auditing.

I'm fine with sticking to the fully-qualified `org.apache.accumulo` prefix rather than the more abbreviated `accumulo` one.